src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -402,6 +402,28 @@ public boolean hasMeeting(Meeting meeting) {
         return meetings.contains(meeting);
     }
 
+    /**
+     * Returns true if {@code meeting} conflicts with at least one other meeting in StonksBook.
+     * Any meetings specified in {@code meetingsToExclude} are not checked against for conflits.
+     */
+    public boolean hasConflictWithOtherMeetings(Meeting meeting, Meeting... meetingsToExclude) {
+        assert meeting != null;
+        assert meetingsToExclude != null;
+
+        List<Meeting> excludedMeetings = List.of(meetingsToExclude);
+
+        for (Meeting meetingToCheckAgainst : meetings) {
+            if (excludedMeetings.contains(meetingToCheckAgainst)) {
+                continue;
+            }
```

Comment:
I'm not sure what this does, could it be omitted?

Reply:
This just prevents the loop from flagging `meetingToCheckAgainst` as conflicting (if it does) if it's in the excluded list. But anyways, I've updated this method to return a list of conflicting meetings so that I can display it in the command feedback!